### PR TITLE
Feat: Add all function related to note

### PR DESCRIPTION
**Pull request by rahulcs754**
- Note add , delete , move to trash ,move to archeve
- Apply note color, label, priority
- Add filter Functionality by label, date wise , priority wise
- Add Dark themes

### CODE REVIEW

File: src/frontend/Component/Inputarea/Input.js [hunk 1]
```diff
@@ -27,42 +34,93 @@ export const Input = () => {
...
     Setpopup((prev) => ({ ...prev, colors: !prev.colors }));
   };
 
+  function saveHandler(e) {
+    e.preventDefault();
+
+    const find = notes.some((item) => item._id === note._id);
+    if (find) {
+      console.log("update karo eskp");
+      updateNote(note, DispatchUserAuth);
```
Comment (raman1999) [inline src/frontend/Component/Inputarea/Input.js:48]: remove console

**Review by raman1999 — APPROVED** [review]
everything looks good to me , you can merge

**Review by coldpigli — APPROVED** [review]
Everything looks good.

File: src/frontend/Component/Inputarea/Input.js (hunk 1 above)
Comment (rahulcs754) [reply src/frontend/Component/Inputarea/Input.js:48]: done

**Pull request merged**